### PR TITLE
1.0.0 release branch

### PR DESCRIPTION
**Pull request by JamesBurnside**
Release branch for 1.0.0

### CODE REVIEW

File: packages/communication-react/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,78 @@
...
-This log was last generated on Wed, 17 Nov 2021 22:21:27 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Dec 2021 19:41:54 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.0.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.0)
+
+Mon, 06 Dec 2021 19:41:54 GMT
+
+TODO: Release notes here.
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:11]: TODO: we should add our release notes here

File: packages/communication-react/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,78 @@
...
+
+### Changes since 1.0.0-beta.8
+
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.0.0-beta.8..@azure/communication-react_v1.0.0)
+
+  - Fix typo in strings field ([PR #1210](https://github.com/azure/communication-ui-library/pull/1210) by 82062616+prprabhu-ms@users.noreply.github.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:17]: TODO: needs pruned still

File: common/config/beachball/changelog-config.ts [hunk 3]
```diff
@@ -7,7 +7,12 @@ export const config: BeachballConfig = {
     customRenderers: {
       renderHeader,
       renderEntry
-    }
+    },
+    groups: [{
```
Comment (JamesBurnside) [inline common/config/beachball/changelog-config.ts:11]: small change that handily rolls up all packages changelogs into the meta package changelog

**Review by alcail — APPROVED** [review]

File: packages/communication-react/CHANGELOG.md (hunk 2 above)
Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:17]: pruned

File: common/config/beachball/changelog-config.ts [hunk 4]
```diff
@@ -23,8 +28,7 @@ export const config: BeachballConfig = {
       include: "packages/*",
       disallowedChangeTypes: [
         "major",
-        "minor",
-        "patch"
+        "prerelease"
```
[the remark this answers is not in the document]
Comment (JamesBurnside) [reply common/config/beachball/changelog-config.ts:31]: reverting this as it breaks cherry-picking from main currently - we can update this after release

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**